pkg/odo/cli/service/create.go

```diff
@@ -272,7 +321,16 @@ func NewCmdServiceCreate(name, fullName string) *cobra.Command {
 			genericclioptions.GenericRun(o, cmd, args)
 		},
 	}
+
+	if experimental.IsExperimentalModeEnabled() {
+		serviceCreateCmd.Use += fmt.Sprintf(" [flags]\n  %s <operator_type> --crd <crd_name> [service_name] [flags]", o.CmdFullName)
+		serviceCreateCmd.Example += fmt.Sprintf("\n\n") + fmt.Sprintf(createOperatorExample, fullName)
+	}
```

Comment:
why have two `if` checks for experimental mode, https://github.com/openshift/odo/pull/2739/files#diff-47ac2191bd75f4970f6c210349223189R331-R333 can be consolidated here